package.json

```diff
@@ -30,7 +30,7 @@
     "format": "prettier -w .",
     "release": "npm run build && release-it",
     "test": "TS_NODE_PROJECT=tests node --test --test-concurrency=none -r ts-node/register tests/**[!build]/index.test.ts",
-    "test:build": "npm run build && tstyche tests/build/*.tst.ts"
```

Comment:
`.tst.ts` files file picked up by default, or you had something else in mind?

Reply:
Would it make sense adding config file with `testFileMatch: "tests/build/*.tst.ts"`, perhaps?

Reply:
With `testFileMatch` only this directory would be crawled. Otherwise TSTyche crawls the whole CWD and applies filters from command line later. Meaning this `tstyche tests/build/*.tst.ts` is applied after crawling everything.

Reply:
Thanks for explain how the TSTyche works under the hood. For future-proofing, I would set `testFileMatch` to `tests/**/*.tst.ts` and then explicitly scope in the command itself to the specific dir, i.e. `build`.

Reply:
Ah.. Got it. Should be correct now. Or?

Reply:
Yes, this is spot on. Thank you.